### ISSUE TITLE
MikTeX slow to build with warning: "pdflatex: warning: running with administrator privleges"

### ISSUE THREAD

**Issue by yagars** [MiKTeX/miktex#388]
Hello,
I am using MikTeX 2.9 64 bit with TeXstudio. When I try to build any tex file, I get the warning:
"pdflatex: warning: running with administrator privleges"
I can disable the warning by placing --miktex-admin in my PdfLateX command (in TeXstudio) but my file is still slow to compile (30 seconds after the first time building). I have tried this same tex file on multiple computers and I don't see this warning and it builds easily within 10 seconds. Note all the other computers are slower than the one where I have the problem (Dell Precision 5510, see log).

My .tex file also runs just fine if I upload it to overleaf. I have tried uninstalling and reinstalling MikTex various times and the problem is always there. I also see the warning if I build the file directly in TeXworks. I am an administrator on my system. The latest time, I installed MikTex via my university (TU Eindhoven). On other computers running this same MikTeX installation, the build runs fine and fast.

I'm sure you need more info from me but I can tell you you won't be able to reproduce the problem so I won't be able to give an MWE.

Thanks in advance.

[miktex-report.txt](https://github.com/MiKTeX/miktex/files/3721660/miktex-report.txt)

**Comment by edocevoli** [comment]
Please attach all log files (collected in a `.zip` file).

**Comment by yagars** [comment]
Thanks @edocevoli . I'm including everything from C:\ProgramData\MiKTeX\2.9\miktex\log and my thesis.log in a zip file. 
[log.zip](https://github.com/MiKTeX/miktex/files/3722069/log.zip)

**Comment by edocevoli** [comment]
Thank you. We need a more detailed execution trace. Please install and start DebugView: https://docs.microsoft.com/en-us/sysinternals/downloads/debugview

Then open a command prompt window as administrator and run these commands:

```
cd your-project-dir
set MIKTEX_TRACE=error,config,process,access,fndb
pdflatex --miktex-admin your-project-file.tex
```

Please save the DebugView log (<kbd>ctrl</kbd>+<kbd>s</kbd>) and attach it to this issue.

**Comment by yagars** [comment]
Thanks for getting back to me.
I ran DebugView. It took quite some time (646seconds it looks like from the Time column). The uncompressed size is 126MB. Now compressed it is ~10MB. By the way, I also see the same warning but regarding epstopdf: warning: running with administrator privleges.

**Comment by edocevoli** [comment]
Thank you. It seems that you are running an old MiKTeX version. I would recommend that you update. If the problem then still persists, we have to investigate further.

**Comment by yagars** [comment]
I have installed the latest versions (I've tried 64 and 32 bit with full uninstalls between them) as well within the past week. The version which you saw in the log was one installed with a package from my university (that was my latest attempt). I see the same issue with the 64 bit newest version of MikTeX. The 32 bit version would not convert epstopdf. Is there anything else I can try?

**Comment by edocevoli** [comment]
The DebugView log was created by an old MiKTeX version, right?

You can refresh the file name database (both in admin and in user mode) and see if this solves the issue.

**Comment by yagars** [comment]
Yes, that's correct but I've seen the same error with the newest MiKTeX version as well. I refreshed the file name database (the option to restart as administrator is grayed out for me). Unfortunately the issue still remains. Should I uninstall MiKTeX, reinstall the newest 64 bit version and generate the DebugView log once more? If so, should I run the install file as administrator?
![texstudio_2019-10-15_19-19-23](https://user-images.githubusercontent.com/56505688/66853992-cdda5100-ef80-11e9-9855-7e7977ed291c.png)

**Comment by edocevoli** [comment]
Yes, it would help to have a DebugView trace with the latest MiKTeX update.

**Comment by yagars** [comment]
I have the newest version installed now. Now I get the error (which I have had before) below. I deleted the -converted-to.pdf file listed below on purpose to test this specific issue. Now I am unable to convert from eps to pdf.

I'm attaching the log files in one zip and the DebugView trace in another. Thanks again.

Sorry, but epstopdf did not succeed.
The log file hopefully contains the information to get MiKTeX going again:
  C:\Users\ross.savage\AppData\Local\MiKTeX\2.9\miktex\log\epstopdf.log
! Package pdftex.def Error: File `"figures/disturbance_sources/""CPSD_eps_FM"-e
ps-converted-to.pdf' not found: using draft setting.
[miktex_log.zip](https://github.com/MiKTeX/miktex/files/3730928/miktex_log.zip)
[DebugView2.zip](https://github.com/MiKTeX/miktex/files/3730929/DebugView2.zip)

**Comment by edocevoli** [comment]
The `epstopdf` log file is missing. Please zip the directory `C:\Users\ross.savage\AppData\Local\MiKTeX\2.9\miktex\log`. Thanks

**Comment by yagars** [comment]
Ok, I'm pretty sure that's what I did. When I download miktex_log.zip from my previous comment, I see an epstopdf log file in there. I'll attach a new one. I'm sorry I now appear to have 2 issues. I have experienced both of these in the past with various versions of MiKTeX but I know it doesn't make it easier to solve with multiple issues at once :)

[log.zip](https://github.com/MiKTeX/miktex/files/3731066/log.zip)

**Comment by edocevoli** [comment]
The `epstopdf` invocation is wrong:

```
epstopdf "--outfile=figures/disturbance_sources/CPSD_eps_FM-eps-converted-to.pdf figures/disturbance_sources/CPSD_eps_FM.eps"
```

This would be correct:

```
epstopdf "--outfile=figures/disturbance_sources/CPSD_eps_FM-eps-converted-to.pdf" "figures/disturbance_sources/CPSD_eps_FM.eps"
```

**Comment by yagars** [comment]
OK, can you point me to where I can alter this?

**Comment by edocevoli** [comment]
My understanding is that the `epstopdf` invocation is controlled by `epstopdf.sty` of package `oberdiek`.

**Comment by yagars** [comment]
I'm sorry but I don't understand why these modifications need to be made. Doesn't this have something to do with my installation of MiKTeX? This .tex file runs fine on all the other systems I have tried and I have never looked into the invocation of epstopdf.

**Comment by edocevoli** [comment]
You can try this: open a command prompt window and:

```
cd "C:\Users\ross.savage\Desktop\Master MW\Project_internship\Latex_internship_copied_overleaf_13oct"
pdflatex --synctex=1 thesis.tex
```

**Comment by yagars** [comment]
Thanks for your help but now I am getting different errors:
![image](https://user-images.githubusercontent.com/56505688/66895542-5c87b600-eff3-11e9-95cd-b080b88d66f8.png)
Log files are attached. 
I think I will just have to work with overleaf.
[log.zip](https://github.com/MiKTeX/miktex/files/3732856/log.zip)

**Comment by edocevoli** [comment]
The log states that `The remote package repository is outdated`. The CTAN infrastructure is currently inconsistent (due to a maintenance job last night). This should be fixed tomorrow.

**Comment by farzamfallahi** [comment]
Hi every one I get this error:  chktex: security risk: running with elevated privileges chktex: major issue: So far, you have not checked for MiKTeX updates, how can I solve it?
